docs/configuration.md

```diff
@@ -5,18 +5,20 @@ nav_order: 2
 ---
 
 # Configuration
+
 {: .no_toc }
 
 Just the Docs has some specific configuration parameters that can be defined in your Jekyll site's \_config.yml file.
 {: .fs-6 .fw-300 }
 
 ## Table of contents
+
 {: .no_toc .text-delta }
 
 1. TOC
 {:toc}
 
----
+----
 
 View this site's [\_config.yml](https://github.com/just-the-docs/just-the-docs/tree/main/_config.yml) file as an example.
```

Comment:
I think this is slightly out of scope since I didn't originally touch this paragraph - I wonder what the intention of linking it this way was. Perhaps better to link the template doc, but I do think the `_config.yml` isn't misleading per se, especially since it doesn't say anything explicitly about using it as a remote theme. Maybe something to clear up in another PR?

Reply:
I agree that it's out of scope for this PR. I only mentioned it because it was displayed in the context for the diff.